src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Colors color, int radius) {
+        setColor(color);
+        setRadius(radius);
```

Comment:
Don’t use setters here. Fix it everywhere ;)
```
this.color = color;
...
```

Reply:
Linter says that color and radius should be private, and for color you need to use a setter

[ERROR] /Users/bohdan.l/IdeaProjects/jv-oop-advanced/src/main/java/core/basesyntax/Figure.java:4:18: Variable 'color' must be private and have accessor methods. [VisibilityModifier]
[ERROR] /Users/bohdan.l/IdeaProjects/jv-oop-advanced/src/main/java/core/basesyntax/Circle.java:4:16: Variable 'radius' must be private and have accessor methods. [VisibilityModifier]